particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,60 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'TicTacToe.arcs'
+import 'GameSchemas.arcs'
+
+particle Game in './source/Game.js'
+  inout Person humanPlayer
+  inout Person computerPlayer
+  in GameState {gameOver} gameState
+  in Move {Text move} humanMove
+  in Move {Text move} computerMove
+  inout Move move
+  check humanPlayer is private
+  check computerPlayer is private
+  consume root
+    provide playerSlot
+    provide greetingSlot
+    provide cellSlot
+
+particle DisplayHumanGreeting in './source/DisplayGreeting.js'
+  in Person player
+  consume greetingSlot
+
+recipe GameDemoRecipe
+  create #volatile #move as move
+  create #volatile #humanMove as humanMove
+  create #volatile #computerMove as computerMove
+  create #volatile as gameState  
+
+  Game
+    consume root 
+      provide greetingSlot as greeting
+      provide cellSlot as board
+    gameState <- gameState
+    humanPlayer <-> humanPlayer
+    computerPlayer <-> computerPlayer
+    humanMove <- humanMove
+    computerMove <- computerMove
+    move <-> move
+
+  MoveApplier
+    move <- move
+    humanMove <-> humanMove
```

Comment:
I doesn't feel right that MoveApplier has access to human and computer moves. It presumably should only read the "move to be applied". I presume you've done this because of some glitch...